# MetaTiME: Meta-components in Tumor immune MicroEnvironment 
<p align="left"><img src="https://github.com/yi-zhang/MetaTiME/blob/main/docs/source/_static/img/logo.png" width="290" height="240"></p>
![Documentation Status](https://readthedocs.org/projects/metatime/badge/?version=latest)](https://metatime.readthedocs.io/en/latest/?badge=latest)
MetaTiME is a framework to generate data-driven, interpretable, and reproducible gene programs by integrating millions of single cells from hundreds of tumor scRNA-seq data. Applied on large-scale tumor scRNA data with 2.7 million cells, MetaTiME thus utilize the meta-components to automatically annotate cell states for single-cells from tumor microenvironment. 
( * BETA version, Currently under development  :)

## Installation
`pip install git+https://github.com/yi-zhang/MetaTiME.git`

## Interactive tutorials
### MetaTiME-Annotator
[Use MetaTiME to automatically annotate cell states and map signatures ![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/yi-zhang/MetaTiME/blob/main/docs/notebooks/metatime_annotator.ipynb)

## Method 
<p align="left"><img src="https://github.com/yi-zhang/MetaTiME/blob/main/docs/source/_static/img/fig1.png" width="700" height="400"></p>


### Usage
 - [Use MetaTiME to automatically annotate cell states and map signatures](https://github.com/yi-zhang/MetaTiME/blob/main/docs/notebooks/metatime_annotator.ipynb)
 - 


### Reference
Manuscript In Submission. More details will be updated!

### Contact


Yi Zhang, Ph.D.
Research Fellow
Department of Data Science
Dana-Farber Cancer Institute
Harvard University T.H. Chan School of Public Health

yiz [AT] ds.dfci.harvard.edu

@Wings7Spread
